webxdc.d.ts

```diff
@@ -58,6 +71,21 @@ interface Webxdc<T> {
    * @param description short, human-readable description what this update is about. this is shown eg. as a fallback text in an email program.
    */
   sendUpdate(update: SendingStatusUpdate<T>, description: string): void;
+  /**
+   * TODO
+   * @param content
+   * @returns returns a promise that is resolved to true once export is completed or false on error/abort
```

Comment:
Is there a particular reason to not reject the promise instead of resolving to `false`? Because the former is common practice.

Reply:
I don't know, maybe @r10s knows the reason for returning the boolean. one reason could be to be able to distinguish errors from user cancellation.

Reply:
there was no particular reason, https://github.com/webxdc/webxdc_docs/pull/45 has the recent state after recent discussions